test/git_repos_manager.py

```diff
@@ -0,0 +1,25 @@
+from . import git_utils as git
+from .data_manager import DataManager
+
+
+class GitReposManager:
+    def __init__(self, test_data_mgr):
+        self._test_data_mgr: DataManager = test_data_mgr
+        self._repos = {}
+
+    def new_empty_repo(self, name):
+        if name in self._repos:
+            raise ValueError(f"{name} repo already created!")
```

Comment:
`ValueError` is for "invalid" argument values?
Learning new stuff.

Reply:
Yep, it's a builtin exception made for that purpose :)